src/views/Home/components/CLWCreditsDaysLeft/index.js

```diff
@@ -138,19 +140,24 @@ const CLWCreditsDaysLeft = () => {
               {'Day' + (daysFinished === 1 ? '' : 's') + ' Finished'}
             </div>
           </div>
-          <div
-            style={{
-              display: 'flex',
-              flexDirection: 'column',
-              justifyContent: 'center',
-              alignItems: 'center',
-            }}
-          >
-            <div className="label-text" style={{ color: chapelColor }}>
-              {current}
+
+          {required ? (
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'column',
+                justifyContent: 'center',
+                alignItems: 'center',
+              }}
+            >
+              <div className="label-text" style={{ color: chapelColor }}>
+                {current}
+              </div>
+              <div className="entry-text">{'CL&W Credit' + (current === 1 ? '' : 's')}</div>
             </div>
-            <div className="entry-text">{'CL&W Credit' + (current === 1 ? '' : 's')}</div>
-          </div>
+          ) : (
+            ''
+          )}
```

Comment:
Again, why not return null instead of an empty string?
```suggestion
          ) : null
          }
```